client-react/src/pages/app/app-settings/ErrorPages/ErrorPageGridAddEdit.tsx

```diff
@@ -41,19 +44,20 @@ const ErrorPageGridAddEdit: React.FC<ErrorPageGridAddEditProps> = React.memo((pr
     };
   }, []);
 
-  console.log(file); // file just for now till i work on PUT API
-
   return (
-    <form className={addEditFormStyle}>
-      <p id="default-documents-info-message">{t('errorPagesEditMessage')}</p>
-      <Stack>
-        <ErrorPageFileUploader
-          setFile={setFile}
-          fileUploadSuccess={fileUploadSuccess}
-          setFileUploadSuccess={setFileUploadSuccess}></ErrorPageFileUploader>
-      </Stack>
-      <ActionBar id="error-page-edit-footer" primaryButton={actionBarPrimaryButtonProps} secondaryButton={actionBarSecondaryButtonProps} />
-    </form>
+    <>
+      <form className={addEditFormStyle}>
+        <p id="error-pages-info-message">{t('errorPagesEditMessage')}</p>
+        <Stack>
+          <ErrorPageFileUploader setFile={setFile} fileUploadSuccess={fileUploadSuccess} setFileUploadSuccess={setFileUploadSuccess} />
+        </Stack>
+        <ActionBar
+          id="error-page-edit-footer"
+          primaryButton={actionBarPrimaryButtonProps}
+          secondaryButton={actionBarSecondaryButtonProps}
+        />
+      </form>
+    </>
```

Comment:
Remove the fragment (`<>`) since `form` is the only child.

```suggestion
    <form className={addEditFormStyle}>
      <p id="error-pages-info-message">{t('errorPagesEditMessage')}</p>
      <Stack>
        <ErrorPageFileUploader setFile={setFile} fileUploadSuccess={fileUploadSuccess} setFileUploadSuccess={setFileUploadSuccess} />
      </Stack>
      <ActionBar
        id="error-page-edit-footer"
        primaryButton={actionBarPrimaryButtonProps}
        secondaryButton={actionBarSecondaryButtonProps}
      />
    </form>
```